drivers/wifi/nrf700x/Kconfig

```diff
@@ -401,3 +401,6 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config WIFI_NRF700X_SKIP_LOCAL_MAC
+	bool "Suppresses local MAC and reserved IANA Unicast MAC addresses in scan results"
```

Comment:
It would be nice to add a phrase, here on why someone would want to do it. A very simple reasoning.